### ISSUE TITLE
Received points but no payment

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#997]
I did not received payment for https://github.com/yegor256/cactoos/issues/747, just points. The payment not appeared even in debt page; I've received payment for other jobs which were listed in debts page before and after https://github.com/yegor256/cactoos/issues/747 completion, but just points for this job.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by amihaiemil** [comment]
@paulodamaso I also had a similar behaviour here: https://github.com/zerocracy/farm/issues/991

**Comment by paulodamaso** [comment]
@amihaiemil Yes, I saw it before opening this issue, but I think is a different bug because #747 had an abnormal behavior from the very beginning.

**Comment by paulodamaso** [comment]
@yegor256 please take a look, I didn't received this payment yet neither in wallet nor in paypal.

**Comment by yegor256** [comment]
@paulodamaso most likely your rate was zero at the moment of assignment. This fix will make the situation more transparent to a developer: 8220bc35c3ec5b8f6c14e2aa8a4906226fe822d6 and 7013353bc1561a5b15cedf31e53f6f0adeaa044e

**Comment by yegor256** [comment]
@paulodamaso thanks for reporting, hope this won't happen again. In the meantime you can explain this situation to the project architect and ask him to use http://www.zerocracy.com/policy.html#49

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@73a6d26f-063d-463b-838b-650a8ef50052, thread: AsyncFlush-5
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.zerocracy.farm.strict.StrictProject[111] java.lang.IllegalArgumentException: File "blanks/paulodamaso.xml" is not accessible in "PMO"
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): CID: 1301148122, Type: 'Close job', Author: yegor256

**Comment by paulodamaso** [comment]
@yegor256 Thanks for the reply.

**Comment by llorllale** [comment]
@yegor256 according to the cactoos slack chat, I successfully assigned `DEV` and `REV` with rate `$16` to @paulodamaso on april 19th.

**Comment by paulodamaso** [comment]
@llorllale @yegor256 No payment yet on yegor256/cactoos#747 for me.

**Comment by yegor256** [comment]
@paulodamaso this is definitely some bug inside, but I can't catch it since we didn't have logs at that time. We will have more visibility in the future. Sorry.

**Comment by llorllale** [comment]
@yegor256 FYI I just manually instructed 0crat to pay @paulodamaso 30min for yegor256/cactoos#747

**Comment by denis-konovalyenko** [comment]
@g4s8 , @yegor256 , @paulodamaso , it seems that this has happened again in the scope of [my](https://github.com/stryvers/life-lens-app/pull/57) issue or I have missed something...

**Comment by g4s8** [comment]
@denis-konovalyenko I don't have an access to your repo, can you say me 0crat project id please, and I'll be able to check it?

**Comment by denis-konovalyenko** [comment]
@g4s8 , this is GFV03LPPY.

**Comment by denis-konovalyenko** [comment]
@g4s8 , @yegor256 , @paulodamaso , think I missed the message from 0crat:

>Zerocrat, [08.03.19 23:49]
>We just paid you $3.68 (90a7d516-b2c4-44b4-a148-5b4d3c79d3af) for #57 (https://github.com/stryvers/life-lens-app/issues/57): Order was finished, quality is "acceptable"the tuition fee $0.32 was deducted and sent to @vatavuk/z (https://www.0crat.com/u/vatavuk) (your mentor), according to §45 (http://www.zerocracy.com/policy.html#45)

Sorry for the buzz.

**Comment by amihaiemil** [comment]
@g4s8 This is not the last time people are going to ask about "missed" (true or not) payments. Maybe it would be wise to have a "Check Payment Status" functionality in 0crat? So people enter the project id and Issue number and see what exactly happened to said payment. 

It's usually hard to link payments ot task, especially if 0crat pays the debt right after the payment (then you have to go to Paypal, check transaction description, check Issue id etc).

**Comment by g4s8** [comment]
@denis-konovalyenko WTS claims that it received the payment, see https://wts.zold.io/output?id=90a7d516-b2c4-44b4-a148-5b4d3c79d3af also you can check it in Telegram channel: https://t.me/zold_wts (just use search with `gh:stryvers/life-lens-app-57`). Try to pull your wallet and check again, if you're not able to find this transaction in your wallet, please submit an issue to https://github.com/zold-io/wts.zold.io/issues

**Comment by g4s8** [comment]
@amihaiemil I think it's a good idea, can you submit a ticket please?

**Comment by denis-konovalyenko** [comment]
@g4s8 , thank you, I have found the transaction.

**Comment by denis-konovalyenko** [comment]
@g4s8 , do you think you would be able to clarify one more case? I am quite sure there was no monetary reward for that - no message from 0crat in Telegram, no transaction in my wallet [e560c4fb95684c6d](http://www.zold.io/ledger.html?wallet=e560c4fb95684c6d).

Telegram message from 0crat:
> Zerocrat, [15.03.19 13:26]
You got +15 point(s) in the job #101 (https://github.com/stryvers/life-lens-app/issues/101) in GFV03LPPY, your total is +340 (https://www.0crat.com/u/denis-konovalyenko/awards), see §18 (http://www.zerocracy.com/policy.html#18): Bug was reported, see §29 (http://www.zerocracy.com/policy.html#29)

More details on the problem:
For some reasons when the issue was put in scope by adding the scope label it did not entered the scope. That happend only after "@ 0crat in" command was issued.

Thank you!

**Comment by g4s8** [comment]
@denis-konovalyenko sorry, but I didn't get your problem: is it about "scope label is not working" or you didn't receive the payment for job?

**Comment by denis-konovalyenko** [comment]
@g4s8 , I did not receive the payment first of all. :) As for the "scope label is not working", I can post a new issue if there is no such.

**Comment by g4s8** [comment]
@denis-konovalyenko as I can see it was added to WBS successfully:
![scrot-2019-03-16-09:27:23](https://user-images.githubusercontent.com/1524988/54471681-e6597180-47cd-11e9-9ab4-ca262c8ff684.png)

you didn't receive the payment because this project doesn't have enough funds to pay you.

**Comment by denis-konovalyenko** [comment]
@g4s8 , thank you very much! Could you please point me out to the place where I can also observer such information for the future? By the way, my WBS page is just listing me my project IDs I am working on: `C3NDPUA8L, GFCHY7NQG, GFV03LPPY`. Is it expected to show more details?

**Comment by g4s8** [comment]
@denis-konovalyenko you can go to profile page https://www.0crat.com/me and see all projects where you have any role, then you can click on each project to see more details, such as available cash, if you see this warning message, it means that project can't pay you:
![scrot-2019-03-16-09:41:38](https://user-images.githubusercontent.com/1524988/54471785-fa9e6e00-47cf-11e9-9ea8-4e194a051e03.png)

**Comment by denis-konovalyenko** [comment]
@g4s8 , thank you for the clarification! Sorry, I have not found the message you shown on the project page:
![image](https://user-images.githubusercontent.com/15904097/54471919-629d8600-47c9-11e9-9b00-97bceff15e11.png)

**Comment by g4s8** [comment]
@denis-konovalyenko did you check this project https://www.0crat.com/p/GFV03LPPY , right?
I can see this warning, maybe you don't have enough permissions?

**Comment by denis-konovalyenko** [comment]
@g4s8 , you are right. This is the correct link to the project. As you can see I have DEV and REV roles. I have no clue what permissions have to be possessed with these roles.

**Comment by denis-konovalyenko** [comment]
@g4s8 , could you please clarify the time frame the claims are usually paid in (the project has been properly funded for some days already)? Thanks!
